Dockerfile

```
@@ -1,20 +1,24 @@
-ARG NODE_IMAGE=node:12.2-alpine
+ARG NODE_IMAGE=node:12.11-alpine
```

So turns there is no tag for v12.11 of NodeJS, yet, as there seems to be an issue on NoeJS/Docker side for building the docker images. See pending pull request https://github.com/nodejs/docker-node/pull/1113

However, what I suggest is using the major version only tag as the base of this image. Doing so also saves us having to update our `Dockerfile` for every minor version released. Meaning every time one builds the docker image for Flood, the latest version of node will be used as the base (if you pull while building `docker build --pull` option. Minor version theoretically not have any BC and if anything does stop working, that should warrant a fix in the app anyway as someone maybe running that version.

```suggestion
ARG NODE_IMAGE=node:12-alpine
```